lib/linguist/languages.yml

```diff
@@ -3372,6 +3372,7 @@ Python:
   color: "#3572A5"
   extensions:
   - ".py"
+  - ".py3"
```

Comment:
Yeah, but not in https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language